Build failure: portmod

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

nix-build -A portmod

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/311412033

### Relevant log output

```console
nix-build -A portmod
these 3 derivations will be built:
  /nix/store/gz83zfxj6w1p6gksrcz4alww9km2abbb-portmod-rust-2.8.1.drv
  /nix/store/qn1fp28zpjbpmfppibnn0mmjhkj9samm-python3.13-redbaron-0.9.2.drv
  /nix/store/dxwffbn5yr0zxcvlnvb6ccy01cmb4mm4-portmod-2.8.1.drv
these 40 paths will be fetched (78.41 MiB download, 628.43 MiB unpacked):
  /nix/store/6q579x9sqd7y153f39gw936f9rdryhba-auditable-cargo-1.89.0
  /nix/store/q7w8anfi759f9jnxnh7alywm4nfx5ywc-bubblewrap-0.11.0-dev
  /nix/store/7q8xx8v6i40wm6w39agdq380p1g6qjgs-bullet-3.25
  /nix/store/ynhfvh8h1da3l2hnm15ab2fh3h8a6lf4-cargo-1.89.0
  /nix/store/jbax1j60561siini5xyfs6j8sb32slvd-cargo-auditable-0.6.5
  /nix/store/z36i0wgls1b9bmc4wczbpbxbx6mbqadi-cargo-build-hook.sh
  /nix/store/3rs36yd4isig8gs1j42dr0rxlmm6m9yi-cargo-check-hook.sh
  /nix/store/3ynp453h5yzyf857dwj69jd4a05wvl8h-cargo-install-hook.sh
  /nix/store/g33mqbx5kpcn0f090k45yll1jiaz5dgn-cargo-setup-hook.sh
  /nix/store/ml0cxapc0dhaa8zbyrs9nk86aailf5fm-collada-dom-2.5.2
  /nix/store/xl2xnhlg8dbvgpazzfy3c4kvg2fziq7z-gnustep-base-1.29.0-lib
  /nix/store/7lcb9pq7n218rslkcbfv8s2l5lk038db-gnustep-libobjc-2.3
  /nix/store/cr2bwiqvdkx6icr2a6cyd5hnk9k7zq73-gnustep-make-2.9.3
  /nix/store/c64v6zn0617brf74nz4r62m5dbk06v7k-lzip-1.25
  /nix/store/drvzj3rp41s6d7g1b0vvara3svvr4azi-mygui-3.4.3
  /nix/store/qgkfx8rhilykcln2lp62674pajlqb38q-openmw-0.49.0
  /nix/store/njlq0cj5ib9xjrjbrcsq1qn26f01jqij-openscenegraph-3.6.5
  /nix/store/r3v6vjcvjbrv69izr6plw5wjjgiv0f90-portmod-rust-2.8.1-vendor
  /nix/store/iwkpibdn90zpy009r4irrs1fih1npyxz-python3.13-chardet-5.2.0
  /nix/store/6ap211dg7z2nf432chfzl8df7863f1s2-python3.13-ddt-1.7.2
  /nix/store/vjd9brsljwqp0lk70n46w2qmdr9zxsqb-python3.13-distlib-0.4.0
  /nix/store/vpzivwnawmsry9jq6c7n75ah2b23p955-python3.13-fasteners-0.20
  /nix/store/03nd16p46h58ymlspav1v21dgkqicpjq-python3.13-gitdb-4.0.12
  /nix/store/6prmlrvp4wndz0s5a0q7m68dd5q29vxv-python3.13-gitpython-3.1.45
  /nix/store/82cq1sjazsrc31agr1sbpf1571z54yny-python3.13-patool-4.0.1
  /nix/store/ixg9jmrwksw3l8z1yi02a3jlhr8l0fi2-python3.13-progressbar2-4.5.0
  /nix/store/3vzbj7vj2nab4bzvgwp7yk8gjv17rqi4-python3.13-pypblib-0.0.4
  /nix/store/9d3jzqmmh76m1s8y4rym8j4ay6pqmwmq-python3.13-python-sat-0.1.8.dev20
  /nix/store/b765x1vicxmnabv3frxlg2bawpz0zri3-python3.13-python-utils-3.9.1
  /nix/store/6db1cfwl68j96f49schj5diqn283mvz1-python3.13-restrictedpython-8.1
  /nix/store/r2as5msxhr0cl9aic9jkq40jzjhg40f9-python3.13-setuptools-scm-9.0.1
  /nix/store/hkfsmbmg8f43kbkh14r6rgqlrac4igxm-python3.13-smmap-6.0.0
  /nix/store/60ic7b8vgbayn31yqqch0264zv09dgf8-python3.13-virtualenv-20.33.1
  /nix/store/apv4vfxbp8a3jprkc3l4vnpv2vrpdizv-source
  /nix/store/4fdzygbmg6ipr3nl66vd05ajbl32kpib-tes3cmd-0.40-pre-release-2
  /nix/store/436zxbkrzx3qfsmzn0gd33fwsmlcxgi6-tr-patcher-1.0.5
  /nix/store/syhqpz616vnnzrvsgyrllzrlm2299rcf-unar-1.10.8
  /nix/store/7pf8mgw6rd5849fsw9nw4mapj8vgbpz4-unshield-1.6.2
  /nix/store/nxzb087pyzn6bx1aqmk1bajsfg7issl7-yaml-cpp-0.8.0
  /nix/store/zwn2y6j0z9xdimk3gxpfliq1xbg50l2x-zpaq-7.15
copying path '/nix/store/apv4vfxbp8a3jprkc3l4vnpv2vrpdizv-source' from 'https://cache.nixos.org'...
copying path '/nix/store/7q8xx8v6i40wm6w39agdq380p1g6qjgs-bullet-3.25' from 'https://cache.nixos.org'...
copying path '/nix/store/q7w8anfi759f9jnxnh7alywm4nfx5ywc-bubblewrap-0.11.0-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/iwkpibdn90zpy009r4irrs1fih1npyxz-python3.13-chardet-5.2.0' from 'https://cache.nixos.org'...
copying path '/nix/store/vpzivwnawmsry9jq6c7n75ah2b23p955-python3.13-fasteners-0.20' from 'https://cache.nixos.org'...
copying path '/nix/store/6db1cfwl68j96f49schj5diqn283mvz1-python3.13-restrictedpython-8.1' from 'https://cache.nixos.org'...
copying path '/nix/store/r2as5msxhr0cl9aic9jkq40jzjhg40f9-python3.13-setuptools-scm-9.0.1' from 'https://cache.nixos.org'...
copying path '/nix/store/4fdzygbmg6ipr3nl66vd05ajbl32kpib-tes3cmd-0.40-pre-release-2' from 'https://cache.nixos.org'...
copying path '/nix/store/436zxbkrzx3qfsmzn0gd33fwsmlcxgi6-tr-patcher-1.0.5' from 'https://cache.nixos.org'...
copying path '/nix/store/ml0cxapc0dhaa8zbyrs9nk86aailf5fm-collada-dom-2.5.2' from 'https://cache.nixos.org'...
copying path '/nix/store/c64v6zn0617brf74nz4r62m5dbk06v7k-lzip-1.25' from 'https://cache.nixos.org'...
copying path '/nix/store/drvzj3rp41s6d7g1b0vvara3svvr4azi-mygui-3.4.3' from 'https://cache.nixos.org'...
copying path '/nix/store/vjd9brsljwqp0lk70n46w2qmdr9zxsqb-python3.13-distlib-0.4.0' from 'https://cache.nixos.org'...
copying path '/nix/store/3vzbj7vj2nab4bzvgwp7yk8gjv17rqi4-python3.13-pypblib-0.0.4' from 'https://cache.nixos.org'...
copying path '/nix/store/b765x1vicxmnabv3frxlg2bawpz0zri3-python3.13-python-utils-3.9.1' from 'https://cache.nixos.org'...
copying path '/nix/store/6ap211dg7z2nf432chfzl8df7863f1s2-python3.13-ddt-1.7.2' from 'https://cache.nixos.org'...
building '/nix/store/qn1fp28zpjbpmfppibnn0mmjhkj9samm-python3.13-redbaron-0.9.2.drv'...
copying path '/nix/store/jbax1j60561siini5xyfs6j8sb32slvd-cargo-auditable-0.6.5' from 'https://cache.nixos.org'...
copying path '/nix/store/ynhfvh8h1da3l2hnm15ab2fh3h8a6lf4-cargo-1.89.0' from 'https://cache.nixos.org'...
copying path '/nix/store/z36i0wgls1b9bmc4wczbpbxbx6mbqadi-cargo-build-hook.sh' from 'https://cache.nixos.org'...
copying path '/nix/store/3rs36yd4isig8gs1j42dr0rxlmm6m9yi-cargo-check-hook.sh' from 'https://cache.nixos.org'...
copying path '/nix/store/3ynp453h5yzyf857dwj69jd4a05wvl8h-cargo-install-hook.sh' from 'https://cache.nixos.org'...
copying path '/nix/store/g33mqbx5kpcn0f090k45yll1jiaz5dgn-cargo-setup-hook.sh' from 'https://cache.nixos.org'...
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
copying path '/nix/store/7lcb9pq7n218rslkcbfv8s2l5lk038db-gnustep-libobjc-2.3' from 'https://cache.nixos.org'...
Sourcing setuptools-build-hook
Using setuptoolsBuildPhase
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing pytest-check-hook
Using pytestCheckPhase
copying path '/nix/store/cr2bwiqvdkx6icr2a6cyd5hnk9k7zq73-gnustep-make-2.9.3' from 'https://cache.nixos.org'...
copying path '/nix/store/r3v6vjcvjbrv69izr6plw5wjjgiv0f90-portmod-rust-2.8.1-vendor' from 'https://cache.nixos.org'...
Running phase: unpackPhase
copying path '/nix/store/hkfsmbmg8f43kbkh14r6rgqlrac4igxm-python3.13-smmap-6.0.0' from 'https://cache.nixos.org'...
unpacking source archive /nix/store/jvnjw715rxqlqaw39zbkddxcmwkg60yv-redbaron-0.9.2.tar.gz
copying path '/nix/store/60ic7b8vgbayn31yqqch0264zv09dgf8-python3.13-virtualenv-20.33.1' from 'https://cache.nixos.org'...
source root is redbaron-0.9.2
setting SOURCE_DATE_EPOCH to timestamp 1552849137 of file "redbaron-0.9.2/setup.cfg"
Running phase: patchPhase
copying path '/nix/store/xl2xnhlg8dbvgpazzfy3c4kvg2fziq7z-gnustep-base-1.29.0-lib' from 'https://cache.nixos.org'...
Running phase: updateAutotoolsGnuConfigScriptsPhase
copying path '/nix/store/7pf8mgw6rd5849fsw9nw4mapj8vgbpz4-unshield-1.6.2' from 'https://cache.nixos.org'...
Running phase: configurePhase
copying path '/nix/store/nxzb087pyzn6bx1aqmk1bajsfg7issl7-yaml-cpp-0.8.0' from 'https://cache.nixos.org'...
copying path '/nix/store/zwn2y6j0z9xdimk3gxpfliq1xbg50l2x-zpaq-7.15' from 'https://cache.nixos.org'...
copying path '/nix/store/ixg9jmrwksw3l8z1yi02a3jlhr8l0fi2-python3.13-progressbar2-4.5.0' from 'https://cache.nixos.org'...
no configure script, doing nothing
Running phase: buildPhase
Executing setuptoolsBuildPhase
setup.py build flags: ''
copying path '/nix/store/9d3jzqmmh76m1s8y4rym8j4ay6pqmwmq-python3.13-python-sat-0.1.8.dev20' from 'https://cache.nixos.org'...
copying path '/nix/store/03nd16p46h58ymlspav1v21dgkqicpjq-python3.13-gitdb-4.0.12' from 'https://cache.nixos.org'...
copying path '/nix/store/syhqpz616vnnzrvsgyrllzrlm2299rcf-unar-1.10.8' from 'https://cache.nixos.org'...
warning: pypandoc module not found, could not convert Markdown to RST
/nix/store/a53syg4q2wk1jhw24gqnib37pkgacndy-python3.13-setuptools-80.9.0/lib/python3.13/site-packages/setuptools/dist.py:759: SetuptoolsDeprecationWarning: License classifiers are deprecated.
!!

        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************

!!
  self._finalize_license_expression()
running bdist_wheel
The [wheel] section is deprecated. Use [bdist_wheel] instead.
/nix/store/a53syg4q2wk1jhw24gqnib37pkgacndy-python3.13-setuptools-80.9.0/lib/python3.13/site-packages/setuptools/_distutils/cmd.py:135: SetuptoolsDeprecationWarning: bdist_wheel.universal is deprecated
!!

        ********************************************************************************
        With Python 2.7 end-of-life, support for building universal wheels
        (i.e., wheels that support both Python 2 and Python 3)
        is being obviated.
        Please discontinue using this option, or if you still need it,
        file an issue with pypa/setuptools describing your use case.

        This deprecation is overdue, please update your project and remove deprecated
        calls to avoid build errors in the future.
        ********************************************************************************

!!
  self.finalize_options()
running build
running build_py
creating build/lib/redbaron
copying redbaron/nodes.py -> build/lib/redbaron
copying redbaron/utils.py -> build/lib/redbaron
copying redbaron/redbaron.py -> build/lib/redbaron
copying redbaron/private_config.py -> build/lib/redbaron
copying redbaron/__init__.py -> build/lib/redbaron
copying redbaron/base_nodes.py -> build/lib/redbaron
copying redbaron/syntax_highlight.py -> build/lib/redbaron
/nix/store/a53syg4q2wk1jhw24gqnib37pkgacndy-python3.13-setuptools-80.9.0/lib/python3.13/site-packages/setuptools/_distutils/cmd.py:90: SetuptoolsDeprecationWarning: setup.py install is deprecated.
!!

        ********************************************************************************
        Please avoid running ``setup.py`` directly.
        Instead, use pypa/build, pypa/installer or other
        standards-based tools.

        By 2025-Oct-31, you need to update your project and remove deprecated calls
        or your builds will no longer be supported.

        See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
        ********************************************************************************

!!
  self.initialize_options()
copying path '/nix/store/82cq1sjazsrc31agr1sbpf1571z54yny-python3.13-patool-4.0.1' from 'https://cache.nixos.org'...
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/redbaron
copying build/lib/redbaron/nodes.py -> build/bdist.linux-x86_64/wheel/./redbaron
copying build/lib/redbaron/utils.py -> build/bdist.linux-x86_64/wheel/./redbaron
copying build/lib/redbaron/redbaron.py -> build/bdist.linux-x86_64/wheel/./redbaron
copying build/lib/redbaron/private_config.py -> build/bdist.linux-x86_64/wheel/./redbaron
copying build/lib/redbaron/__init__.py -> build/bdist.linux-x86_64/wheel/./redbaron
copying build/lib/redbaron/base_nodes.py -> build/bdist.linux-x86_64/wheel/./redbaron
copying build/lib/redbaron/syntax_highlight.py -> build/bdist.linux-x86_64/wheel/./redbaron
running install_egg_info
running egg_info
writing redbaron.egg-info/PKG-INFO
writing dependency_links to redbaron.egg-info/dependency_links.txt
writing requirements to redbaron.egg-info/requires.txt
writing top-level names to redbaron.egg-info/top_level.txt
reading manifest file 'redbaron.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'redbaron.egg-info/SOURCES.txt'
Copying redbaron.egg-info to build/bdist.linux-x86_64/wheel/./redbaron-0.9.2-py3.13.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/redbaron-0.9.2.dist-info/WHEEL
creating 'dist/redbaron-0.9.2-py2.py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'redbaron/__init__.py'
adding 'redbaron/base_nodes.py'
adding 'redbaron/nodes.py'
adding 'redbaron/private_config.py'
adding 'redbaron/redbaron.py'
adding 'redbaron/syntax_highlight.py'
adding 'redbaron/utils.py'
adding 'redbaron-0.9.2.dist-info/METADATA'
adding 'redbaron-0.9.2.dist-info/WHEEL'
adding 'redbaron-0.9.2.dist-info/top_level.txt'
adding 'redbaron-0.9.2.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
copying path '/nix/store/6prmlrvp4wndz0s5a0q7m68dd5q29vxv-python3.13-gitpython-3.1.45' from 'https://cache.nixos.org'...
Finished executing setuptoolsBuildPhase
Running phase: installPhase
Executing pypaInstallPhase
copying path '/nix/store/njlq0cj5ib9xjrjbrcsq1qn26f01jqij-openscenegraph-3.6.5' from 'https://cache.nixos.org'...
copying path '/nix/store/6q579x9sqd7y153f39gw936f9rdryhba-auditable-cargo-1.89.0' from 'https://cache.nixos.org'...
/nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2/lib/python3.13/site-packages/redbaron/base_nodes.py:1256: SyntaxWarning: invalid escape sequence '\s'
  if re.match("^\s*%s" % indented_type, string):
/nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2/lib/python3.13/site-packages/redbaron/base_nodes.py:690: SyntaxWarning: "is" with 'str' literal. Did you mean "=="?
  if self.on_attribute is "root":
/nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2/lib/python3.13/site-packages/redbaron/base_nodes.py:1256: SyntaxWarning: invalid escape sequence '\s'
  if re.match("^\s*%s" % indented_type, string):
Successfully installed redbaron-0.9.2-py2.py3-none-any.whl
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: fixupPhase
shrinking RPATHs of ELF executables and libraries in /nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2
checking for references to /build/ in /nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2...
patching script interpreter paths in /nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2
stripping (with command strip and flags -S -p) in  /nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2/lib
shrinking RPATHs of ELF executables and libraries in /nix/store/6f973r86g60f9a7ws0nx6r87wxajbj1g-python3.13-redbaron-0.9.2-dist
checking for references to /build/ in /nix/store/6f973r86g60f9a7ws0nx6r87wxajbj1g-python3.13-redbaron-0.9.2-dist...
patching script interpreter paths in /nix/store/6f973r86g60f9a7ws0nx6r87wxajbj1g-python3.13-redbaron-0.9.2-dist
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
Running phase: pythonRemoveBinBytecodePhase
Running phase: pythonImportsCheckPhase
Executing pythonImportsCheckPhase
Running phase: pytestCheckPhase
Executing pytestCheckPhase
pytest flags: -m pytest
============================= test session starts ==============================
platform linux -- Python 3.13.8, pytest-8.4.2, pluggy-1.6.0
rootdir: /build/redbaron-0.9.2
copying path '/nix/store/qgkfx8rhilykcln2lp62674pajlqb38q-openmw-0.49.0' from 'https://cache.nixos.org'...
collected 1431 items

tests/test_at.py .                                                       [  0%]
tests/test_helpers.py ..............................                     [  2%]
tests/test_indentation.py ....                                           [  2%]
tests/test_initial_parsing.py .......................................... [  5%]
........................................................................ [ 10%]
...................................                                      [ 12%]
tests/test_insert.py .............................                       [ 14%]
tests/test_no_pygments.py ........F                                      [ 15%]
tests/test_path.py ............................                          [ 17%]
tests/test_position.py ............................                      [ 19%]
tests/test_proxy_list.py ............................................... [ 22%]
........................................................................ [ 27%]
...........................................................              [ 31%]
tests/test_redbaron.py .....                                             [ 32%]
tests/test_regressions.py ....                                           [ 32%]
tests/test_render.py ....                                                [ 32%]
tests/test_root.py ..                                                    [ 32%]
tests/test_setter.py ................................................... [ 36%]
........................................................................ [ 41%]
........................................................................ [ 46%]
........................................................................ [ 51%]
........................................................................ [ 56%]
........................................................................ [ 61%]
........................................................................ [ 66%]
........................................................................ [ 71%]
........................................................................ [ 76%]
........................................................................ [ 81%]
........................................................................ [ 86%]
........................................................................ [ 91%]
........................................................................ [ 96%]
.............................................                            [100%]

=================================== FAILURES ===================================
________________________________ test_highlight ________________________________

capsys = <_pytest.capture.CaptureFixture object at 0x7ffff57db8c0>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x7ffff4917a80>

    def test_highlight(capsys, monkeypatch):
        try:
            import pygments  # noqa
        except ImportError:
            pytest.skip('missing pygments.')
        monkeypatch.setattr(redbaron, 'force_ipython_behavior', True)

        RedBaron("a = []").help()
        captured = capsys.readouterr()

        if tuple(map(int, pytest.__version__.split('.'))) <= (3, 3):
            out = captured[0]
        else:
            out = captured.out
>       assert out == """\
    0 -----------------------------------------------------
    \x1b[38;5;148mAssignmentNode\x1b[39m\x1b[38;5;197m(\x1b[39m\x1b[38;5;197m)\x1b[39m
    \x1b[38;5;15m  \x1b[39m\x1b[38;5;242m# identifiers: assign, assignment, assignment_, assignmentnode\x1b[39m
    \x1b[38;5;15m  \x1b[39m\x1b[38;5;15moperator\x1b[39m\x1b[38;5;197m=\x1b[39m\x1b[38;5;186m''\x1b[39m
    \x1b[38;5;15m  \x1b[39m\x1b[38;5;15mtarget\x1b[39m\x1b[38;5;15m \x1b[39m\x1b[38;5;197m->\x1b[39m
    \x1b[38;5;15m    \x1b[39m\x1b[38;5;148mNameNode\x1b[39m\x1b[38;5;197m(\x1b[39m\x1b[38;5;197m)\x1b[39m
    \x1b[38;5;15m      \x1b[39m\x1b[38;5;242m# identifiers: name, name_, namenode\x1b[39m
    \x1b[38;5;15m      \x1b[39m\x1b[38;5;15mvalue\x1b[39m\x1b[38;5;197m=\x1b[39m\x1b[38;5;186m'a'\x1b[39m
    \x1b[38;5;15m  \x1b[39m\x1b[38;5;15mannotation\x1b[39m\x1b[38;5;15m \x1b[39m\x1b[38;5;197m->\x1b[39m
    \x1b[38;5;15m    \x1b[39m\x1b[38;5;186mNone\x1b[39m
    \x1b[38;5;15m  \x1b[39m\x1b[38;5;15mvalue\x1b[39m\x1b[38;5;15m \x1b[39m\x1b[38;5;197m->\x1b[39m
    \x1b[38;5;15m    \x1b[39m\x1b[38;5;15m[\x1b[39m\x1b[38;5;15m]\x1b[39m
    """
E       AssertionError: assert '0 ----------...39m\x1b[39m\n' == '0 ----------...5m]\x1b[39m\n'
E
E         Skipping 83 identical leading characters in diff, use -v to show
E         - 39m()
E         ?           ^^^              ^^^
E         + 39m()
E         ?           ^^^              ^^^
E         -   # identifiers: assign, assignment, assignment_, assignmentnode...
E
E         ...Full output truncated (35 lines hidden), use '-vv' to show

tests/test_no_pygments.py:56: AssertionError
=============================== warnings summary ===============================
redbaron/base_nodes.py:1256
  /build/redbaron-0.9.2/redbaron/base_nodes.py:1256: SyntaxWarning: invalid escape sequence '\s'
    if re.match("^\s*%s" % indented_type, string):

redbaron/base_nodes.py:690
  /build/redbaron-0.9.2/redbaron/base_nodes.py:690: SyntaxWarning: "is" with 'str' literal. Did you mean "=="?
    if self.on_attribute is "root":

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_no_pygments.py::test_highlight - AssertionError: assert '0 ----------...39m\x1b[39m\n' == '0 ----------...5m...
================== 1 failed, 1430 passed, 2 warnings in 5.09s ==================
error: Cannot build '/nix/store/qn1fp28zpjbpmfppibnn0mmjhkj9samm-python3.13-redbaron-0.9.2.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/6f973r86g60f9a7ws0nx6r87wxajbj1g-python3.13-redbaron-0.9.2-dist
         /nix/store/jx9vj0nsi3z4nhdgwhcmdjdk4lwijc6j-python3.13-redbaron-0.9.2
       Last 25 log lines:
       > E       AssertionError: assert '0 ----------...39m\x1b[39m\n' == '0 ----------...5m]\x1b[39m\n'
       > E
       > E         Skipping 83 identical leading characters in diff, use -v to show
       > E         - 39m()
       > E         ?           ^^^              ^^^
       > E         + 39m()
       > E         ?           ^^^              ^^^
       > E         -   # identifiers: assign, assignment, assignment_, assignmentnode...
       > E
       > E         ...Full output truncated (35 lines hidden), use '-vv' to show
       >
       > tests/test_no_pygments.py:56: AssertionError
       > =============================== warnings summary ===============================
       > redbaron/base_nodes.py:1256
       >   /build/redbaron-0.9.2/redbaron/base_nodes.py:1256: SyntaxWarning: invalid escape sequence '\s'
       >     if re.match("^\s*%s" % indented_type, string):
       >
       > redbaron/base_nodes.py:690
       >   /build/redbaron-0.9.2/redbaron/base_nodes.py:690: SyntaxWarning: "is" with 'str' literal. Did you mean "=="?
       >     if self.on_attribute is "root":
       >
       > -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
       > =========================== short test summary info ============================
       > FAILED tests/test_no_pygments.py::test_highlight - AssertionError: assert '0 ----------...39m\x1b[39m\n' == '0 ----------...5m...
       > ================== 1 failed, 1430 passed, 2 warnings in 5.09s ==================
       For full logs, run:
         nix log /nix/store/qn1fp28zpjbpmfppibnn0mmjhkj9samm-python3.13-redbaron-0.9.2.drv
error: Cannot build '/nix/store/dxwffbn5yr0zxcvlnvb6ccy01cmb4mm4-portmod-2.8.1.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/qgyqf5c3qpd6p92yhvqpss7jdxad5s74-portmod-2.8.1-dist
         /nix/store/ymknjm6rhmnylpknyy0rniq5kjri9j1y-portmod-2.8.1
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.55, NixOS, 25.11 (Xantusia), 25.11.20251028.08dacfc`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - channels(root): `"home-manager, nixos"`
 - nixpkgs: `/nix/store/p6gvrb36n5dh0cdg6d77rkpapy5kc0gq-source`

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/po/portmod/package.nix
+++ b/pkgs/by-name/po/portmod/package.nix
@@ -13,20 +13,20 @@
 }:
 
 let
-  version = "2.8.1";
+  version = "2.9.2";
 
   src = fetchFromGitLab {
     owner = "portmod";
     repo = "Portmod";
-    rev = "v${version}";
-    hash = "sha256-d5XNfjDgtBkNkUMhShYTjKtMbwVa2tLXdvYi6sXQmIA=";
+    rev = "aa454a8ed943758a2f73efa82766d5f627bb3ccd"; # Release 2.9.2 has no upstream tag
+    hash = "sha256-bnCyjHy3I5ETnfvkTGPEwVD5LtrvavU9AisFvQ942l8=";
   };
 
   portmod-rust = rustPlatform.buildRustPackage {
     inherit src version;
     pname = "portmod-rust";
 
-    cargoHash = "sha256-hLci2O+eliCgscvvC4ejn6ZDtFQnM5K6f0luu2cYIHM=";
+    cargoHash = "sha256-KVs1LqwsfWaQb91mhlyKMAh0uDEDDAJHsUSLMIfJq18=";
 
     nativeBuildInputs = [
       python3Packages.python

